orga/tests/integration/components/tables/pagination-control_test.js

```diff
@@ -31,54 +31,62 @@ module('Integration | Component | Table::PaginationControl', function (hooks) {
     this.set('meta', getMetaForPage({ pageNumber: 1 }));
 
     // when
-    await render(hbs`<Table::PaginationControl @pagination={{this.meta}} />`);
+    const screen = await render(hbs`<Table::PaginationControl @pagination={{this.meta}} />`);
 
     // then
-    assert.contains('Page 1 / 2');
+    assert.ok(screen.getByText(this.intl.t('common.pagination.page-number', { current: 1, total: 2 })));
   });
 
   test('it should disable previous button when user is on first page', async function (assert) {
     // given
     this.set('meta', getMetaForPage({ pageNumber: 1 }));
 
     // when
-    await render(hbs`<Table::PaginationControl @pagination={{this.meta}} />`);
+    const screen = await render(hbs`<Table::PaginationControl @pagination={{this.meta}} />`);
 
     // then
-    assert.dom('[aria-label="Aller à la page précédente"]').hasAttribute('disabled');
+    assert
+      .dom(screen.getByRole('button', { name: this.intl.t('common.pagination.action.previous') }))
```

Comment:
**suggestion**
Sur `screen.getByRole('button', { name: this.intl.t('common.pagination.action.previous')`, on doit pouvoir accéder aux attributes. Cela permettrait de se débarrasser de `assert.dom`.